dev-cpp/cpp-taskflow/cpp-taskflow-3.0.0.ebuild

```diff
@@ -12,42 +12,28 @@ SRC_URI="https://github.com/${PN}/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
 LICENSE="MIT"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
-IUSE="doc"
+IUSE="examples test"
+RESTRICT="!test? ( test )"
 
-RDEPEND=""
-DEPEND=""
+PATCHES=( "${FILESDIR}"/${P}-gnuinstalldir.patch )
 
 S="${WORKDIR}/taskflow-${PV}"
-
-PATCHES=(
-)
-
 HTML_DOCS=( docs/. )
 
 src_prepare() {
-	default
-
 	# fix library directoy
 	sed -i "s#/lib#/$(get_libdir)#g" CMakeLists.txt || die "sed failed"
 
-	cmake-utils_src_prepare
+	cmake_src_prepare
 }
```

Comment:
do you still need to override this whole phase function?

Reply:
Actually no since we patched this out, I'll remove it

Reply:
Done